src/components/events/CampusSafetyIncidentReporterWidget.tsx

```
@@ -0,0 +1,317 @@
+      latitude: userCoords.lat,
+      longitude: userCoords.lng,
+      locationLabel: locationLabel.trim(),
+      status: "active",
+    });
+
+    setActiveReport(newReport);
+    if (onReportSubmitted) onReportSubmitted(newReport);
+
+    setShowTriageModal(false);
+    setDispatchSuccess(
+      "High-priority emergency alert dispatched to Organizer Dashboard & Campus PD SMS!"
```

ESLint

[error] Insert `,`

(prettier/prettier)